heyorca/laradock
# Moving Nginx Dockerfile and sites/default.conf

PROBLEM: We need to be able to generate self-signed certs for local deployments

SOLUTION: Move necessary Nginx configs to the repo that will submodule this one to allow it to generate certs and configure Nginx to serve the applicaiton using it.

TO TEST:
- Take previous contents of nginx/Dockerfile and nginx/sites/default.conf and include them in your repo that submodules this one as /nginxDockerfile and /nginxsite.conf
- docker-compose up --build
- confirm nginx container still builds and runs as expected



##### I completed the 3 steps below:

- [] I've read the [Contribution Guide](http://laradock.io/contributing).
- [] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [] I enjoyed my time contributing and making developer's life easier :)
